.github/workflows/deploy.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        docker: [bootstrap, core]
+        docker: [base, bootstrap, core]
```

Comment:
core needs base to build before it, how can we guarantee that ?

Reply:
That's the problem, if you update base the core of the same PR will not use the same image (or will not be available [https://github.com/bluerobotics/companion-docker/pull/227/checks?check_run_id=2521026033#step:7:191])

Reply:
Is it even necessary to include base here, since core is already bringing it into scope?

Reply:
core won't automatically build base. it needs to be built somewhere. I considered doing a [multi-stage dockerfile](https://docs.docker.com/develop/develop-images/multistage-build/),** but I'm afraid that might not fix the caching issues we're having.